### PR TITLE
[JENKINS-43985] - Upgrade Remoting Java requirement to Java 8

### PR DESCRIPTION
**Pull request by oleg-nenashev**
Discussion is here: https://groups.google.com/forum/#!searchin/jenkinsci-dev/remoting%7Csort:date/jenkinsci-dev/CDZaG3dr-kc/YqG4blalAQAJ

https://issues.jenkins-ci.org/browse/JENKINS-43985

@reviewbybees

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -233,7 +233,7 @@ THE SOFTWARE.
           <!--rulesets>
             <ruleset>ruleset.xml</ruleset>
           </rulesets-->
-          <targetJdk>1.6</targetJdk>
+          <targetJdk>1.${java.level}</targetJdk>
```
Comment (svanoort) [inline pom.xml]: Heh

**Comment by jglick** [comment]
Can https://jenkins.ci.cloudbees.com/job/libraries/job/remoting/ be disabled already?

**Comment by oleg-nenashev** [comment]
Reconfigured https://jenkins.ci.cloudbees.com/job/libraries/job/remoting/ to use Java 8. But yes, it should be disabled ASAP (when I fix the PR builder)

**Comment by oleg-nenashev** [comment]
@reviewbybees done

**Comment by oleg-nenashev** [comment]
I will fix the PR builder eventually, works on my machine (c)

**Pull request merged**